Examples/Algorithms/Vertexing/src/AdaptiveMultiVertexFinderAlgorithm.cpp

```diff
@@ -57,6 +57,31 @@ ActsExamples::AdaptiveMultiVertexFinderAlgorithm::
 ActsExamples::ProcessCode
 ActsExamples::AdaptiveMultiVertexFinderAlgorithm::execute(
     const ActsExamples::AlgorithmContext& ctx) const {
+  if (m_cfg.seedFinder == GaussianSeeder) {
+    using Seeder = Acts::TrackDensityVertexFinder<
+        Fitter, Acts::GaussianTrackDensity<Acts::BoundTrackParameters>>;
+    using Finder = Acts::AdaptiveMultiVertexFinder<Fitter, Seeder>;
+    Seeder seedFinder;
+    return executeAfterSeederChoice<Seeder, Finder>(ctx, seedFinder);
+  } else if (m_cfg.seedFinder == AdaptiveGridSeeder) {
+    using Seeder = Acts::AdaptiveGridDensityVertexFinder<109, Fitter>;
+    using Finder = Acts::AdaptiveMultiVertexFinder<Fitter, Seeder>;
+    // The seeder config argument corresponds to the bin size in mm
+    Seeder::Config seederConfig(0.05);
```

Comment:
Is this 0.05 the bin size? can (or should) it be configured?

Reply:
Yes it is the bin size.
It can be configured. I just chose the value the Bastian recommended to me here!